WordPress/wordpress-playground
# Add i18n infrastructure for Playground web app

## Motivation for the change, related issues

We want to make translations possible for the Playground web app.

## Implementation details

This PR:
- [x] Adds a small function for extracting WP-i18n-style gettext strings for translation
- [x] Adds an nx executor for generating a POT file from a codebase
- [x] Adds an i18n module to the Playground website package that exports `@wordpress/i18n` functions configured to use the "wordpress-playground-website" text domain.
- [x] Makes some introductory conversion of hard-coded strings to use the i18n API, and adds a `packages/playground/website/website.pot` file to demonstrate their extraction.

Remaining work
- [ ] Confirm strings for `_n()`, _x()`, and `_nx()` are extracted properly.
- [ ] Dynamically load website translation for declared language when available

## Testing Instructions (or ideally a Blueprint)

- CI
- Smoke test with `npm run dev`
- Manually run `npx nx extract-i18n-strings playground-website` and confirm that `packages/playground/website/website.pot` contains the expected strings.
